src/test/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePluginTest.java

```diff
@@ -50,13 +48,15 @@
 import cloud.fogbow.common.exceptions.UnexpectedException;
 import cloud.fogbow.ras.core.plugins.interoperability.opennebula.OpenNebulaClientUtil;
 
+import static cloud.fogbow.ras.core.plugins.interoperability.opennebula.compute.v5_4.OpenNebulaComputePlugin.*;
+
 @RunWith(PowerMockRunner.class)
```

Comment:
This annotation can be removed, as this class extends **OpenNebulaBaseTests** and, consequently, **BaseUnitTests**.

Reply:
good catch